infrastructure/src/main/java/org/corfudb/infrastructure/log/statetransfer/batchprocessor/BatchProcessorError.java

```diff
@@ -0,0 +1,19 @@
+package org.corfudb.infrastructure.log.statetransfer.batchprocessor;
+
+import lombok.Getter;
+import org.corfudb.infrastructure.log.statetransfer.StateTransferException;
+
+/**
+ * An error that should be handled by the batch processor instance.
+ */
+@Getter
+public class BatchProcessorError extends StateTransferException {
```

Comment:
Please rename to BatchProcessorException
Error - is confusing

Reply:
Was removed.